docs/src/Tools.md

```diff
@@ -204,28 +226,34 @@ The trick, therefore, is to get your host computer to connect to that serial por
 To do that, we a terminal emulator called **screen**.  But first, we have to identify the serial port device that was created when the program started up.
 A simple way to do that is to look in the `/dev/` directory on your computer for a device that includes "usb" in its name:
 
-```
-    ls /dev/*usb*
+```bash
+$ ls /dev/*usb*
 ```
 
-On my machine, this lists two:
+Output on macOS may look like this:
 
+```bash
+/dev/cu.usbmodem14201	/dev/tty.usbmodem14201
 ```
-    /dev/cu.usbmodem14201	/dev/tty.usbmodem14201
+
+On Linux, it may look like this:
+
+```bash
+/dev/cu.usbmodem14201	/dev/tty.usbmodem14201
 ```
 
 To use screen, we specify the first of these devices and a baud rate, as follows:
 
-```
-    screen /dev/cu.usbmodem14201 115200
+```bash
+$ screen <device-name> 115200
```

Comment:
Add:

Where `<device-name>` is the `cu` version, e.g. `/dev/cu.usbmodem14201`.

Reply:
I updated this section. Please review 881f8e0cd971d2ced383fdf02787e249d0890e31.